cloudObjectManager.js

```diff
@@ -0,0 +1,45 @@
+class CloudObjectManager{
+	constructor(){
+	}
+	createProduct(name, cost, imageUrl){
+		const newProduct = new CB.CloudObject("Product");
+		newProduct.set("name", name);
+		newProduct.set("cost_in_cents", cost);
+		newProduct.set("image_url", imageUrl);
+		newProduct.save({
+			success: function(data) {
+				console.log(data);
+			},
+			error: function(err) {
+				if(err == "Unique constraint violated."){
+					console.log("This product already exists.");
+				}
+			}
+		});
+	}
+	findProductsInStore(store){
```

Comment:
We talked about this already, but to document it. The store class should probably have a variable called `this.products = []`.